src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -27,11 +30,15 @@ function PopupWithForm({
           />
         </button>
         <h2 className="form__title">{title}</h2>
-        {children}
-        <p className="form__error">The email is not available</p>
+        {React.Children.map(children, (child) =>
+          React.cloneElement(child, { setIsValid, setValues, values, isOpen })
+        )}
+        <p className="form__error">{signError}</p>
         <button
           onClick={handleSubmit}
```

Comment:
`          onClick={handleSubmit}
`

please, delete it from the submit button


Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses